contracts/modules/cover/CoverViewer.sol

```diff
@@ -127,74 +127,74 @@ contract CoverViewer {
 
   /* ========== COVER PRICING VIEWS ========== */
 
-  function getPoolAllocationPriceParametersForProduct(
-    uint poolId,
-    uint productId
-  ) public view returns (
-    PoolAllocationPriceParameters memory params
-  ) {
-
-    ICover _cover = cover();
-
-    IStakingPool _pool = _cover.stakingPool(poolId);
-    Product memory product = _cover.products(productId);
-
-    uint[] memory staked;
-
-    // FIXME: activeCover is actually allocatedStake
-    // FIXME: I think we want to return the allocated stake instead,
-    // FIXME: because active cover amount will change if the capacity factors are changed
-    (
-    params.activeCover,
-    staked,
-    params.lastBasePrice,
-    params.targetPrice
-    ) = _pool.getPriceParameters(productId, _cover.MAX_COVER_PERIOD());
-
-    params.capacities = new uint[](staked.length);
-
-    for (uint i = 0; i < staked.length; i++) {
-      params.capacities[i] = calculateCapacity(
-        staked[i],
-        product.capacityReductionRatio
-      );
-    }
-
-    params.initialPriceRatio = product.initialPriceRatio;
-  }
-
-  struct PoolAllocationPriceParameters {
-    uint activeCover;
-    uint[] capacities;
-    uint initialPriceRatio;
-    uint lastBasePrice;
-    uint targetPrice;
-  }
-
-  function getPoolAllocationPriceParameters(uint poolId) public view returns (
-    PoolAllocationPriceParameters[] memory params
-  ) {
-    uint count = cover().productsCount();
-    params = new PoolAllocationPriceParameters[](count);
-
-    for (uint i = 0; i < count; i++) {
-      params[i] = getPoolAllocationPriceParametersForProduct(poolId, i);
-    }
-  }
-
-  /* ========== CAPACITY CALCULATION ========== */
-
-  function calculateCapacity(
-    uint staked,
-    uint capacityReductionRatio
-  ) public view returns (uint) {
-
-    ICover _cover = cover();
-    return staked *
-    _cover.globalCapacityRatio() *
-    (CAPACITY_REDUCTION_DENOMINATOR - capacityReductionRatio) /
-    GLOBAL_CAPACITY_DENOMINATOR /
-    CAPACITY_REDUCTION_DENOMINATOR;
-  }
+//  function getPoolAllocationPriceParametersForProduct(
```

Comment:
these are still necessary for the the backend router so we need at least a TODO here to fix things and a clarification.

They are not mandatory for the audit, that is true

Reply:
These functions need an overhaul. Due to changed signatures they started to error out and since they need reimplementation I decided to comment them out for the moment.